packages/rocketchat-api/server/v1/users.js

```diff
@@ -271,6 +271,76 @@ RocketChat.API.v1.addRoute('users.createToken', { authRequired: true }, {
 	}
 });
 
+RocketChat.API.v1.addRoute('users.getPreferences', { authRequired: true }, {
+	get() {
+		const user = this.isUserFromParams() ? RocketChat.models.Users.findOneById(this.userId) : this.getUserFromParams();
+		if (user.settings) {
+			const preferences = user.settings.preferences;
+			preferences['language'] = user.language;
+
+			return RocketChat.API.v1.success({
+				preferences
+			});
+		} else {
+			return RocketChat.API.v1.failure(TAPi18n.__('Accounts_Default_User_Preferences').toUpperCase());
```

Comment:
This error message does not make sense to me

Reply:
@rodrigok what would you put in stead?

Reply:
I don't think that message really to mean would indicate an error.  Needs to be something more along lines of "Failed to do X because of Y"  in my opinion anyways :)

Reply:
@geekgonecrazy that would be a good approach. What you reckon @rodrigok ??
I'm open to use any wording really, just don't feel very creative about tokenization here.
Any suggestion??